Podfile

```diff
@@ -7,6 +7,17 @@ ENV['SWIFT_VERSION'] = '5'
 
 workspace 'Airwallex.xcworkspace'
 
+def shared_profiling_pods
+pod 'AirwallexBehavioralBiometrics', path: 'TrustDefender'
```

Comment:
nitpicking: missing space in front

Reply:
Whoops, that's an oversight. Not a nitpick at all!

Reply:
Whoops, that's an oversight. Not a nitpick at all!